SystemInfo/Value/ComposerPackage.php

```diff
@@ -34,6 +34,16 @@ class ComposerPackage extends ValueObject implements SystemInfo
     public $version;
 
     /**
+     * Stability.
+     *
+     * String with integer, as set in composer.lock "stability-flags".
```

Comment:
Sorry, wrong review order :-)

P.S. if you want to avoid complicated testing, I suggest you use a dedicated service for parsing the stability.